### ISSUE TITLE
Can we have support for postgres macaddr8 as a datatype?

### ISSUE THREAD

**Issue by anthonyorona** [sequelize/sequelize#13644]
Postgres has a datatype for storage of mac addresses in EUI-64 format

https://www.postgresql.org/docs/10/datatype-net-types.html

This would be nice to have without need to create a sequelize addition.

**Comment by WikiRik** [comment]
I think that is a good addition. I've made a [PR](https://github.com/validatorjs/validator.js/pull/1865) for EUI-64 format validation in validator.js. Once that is merged and a version with that is released I will create a PR here to add the support.

**Comment by ephys** [comment]
While a PR for this is welcome, `macaddr8` can already be used today:

```typescript
User = sequelize.define('User', {
  macaddr: {
    type: 'MACADDR8',
  },
});
```
